src/ucs/gtest/main.cc

```diff
@@ -37,5 +50,13 @@ int main(int argc, char **argv) {
     }
     UCS_TEST_MESSAGE << "Using random seed of " << ucs_gtest_random_seed;
     srand(ucs_gtest_random_seed);
+    if (RUNNING_ON_VALGRIND) {
+        modify_config_for_valgrind("IB_RX_QUEUE_LEN", "512");
+        modify_config_for_valgrind("IB_RX_BUFS_GROW", "512");
+        modify_config_for_valgrind("MM_RX_BUFS_GROW", "32");
```

Comment:
not sure. i believe you have several memory pools there
and what about queue lengths?
can you run ucx_info -c on a machine with ugni to collect all variables you want to change here?